examples/training/semantic_segmentation/pascal_voc/deeplab_v3.py

```diff
@@ -169,4 +181,10 @@ def proc_eval_fn(examples):
 ]
 model.compile(optimizer=optimizer, loss=loss_fn, metrics=metrics)
 
-model.fit(train_ds, epochs=FLAGS.epochs, validation_data=eval_ds, callbacks=callbacks)
+model.fit(
+    train_ds,
+    epochs=FLAGS.epochs,
+    validation_data=eval_ds,
+    callbacks=callbacks,
+    #verbose=2,
```

Comment:
remove?

Reply:
Isn't present in the master branch, this was for testing (verbose='AUTO' on Kaggle gets ugly)